admin/linux/build-appimage.sh

```diff
@@ -35,7 +37,28 @@ cd build
 cmake -G Ninja -D CMAKE_INSTALL_PREFIX=/usr ..
 cmake --build . --target all
 DESTDIR=/app cmake --install .
+cd ../..
+
+# KArchive
+git clone https://invent.kde.org/frameworks/extra-cmake-modules.git
+cd extra-cmake-modules
+git checkout v$VER_KFRAMEWORKS
+mkdir build
+cd build
+cmake -G Ninja ..
+cmake --build . --target all
+cmake --install .
+cd ../..
 
+git clone https://invent.kde.org/frameworks/karchive.git
+cd karchive
+git checkout v$VER_KFRAMEWORKS
+mkdir build
+cd build
+cmake -G Ninja -D CMAKE_INSTALL_PREFIX=/usr ..
+cmake --build . --target all
+DESTDIR=/app cmake --install .
+cd ../..
```

Comment:
no need to build them here
they are already included in the AppImage build image

Reply:
done